### DISCUSSION TITLE
only one of three rp2060 can work with the same program

### DISCUSSION THREAD

**Discussion by proxler** [micropython/micropython#16385] [Using MicroPython]
Hallo! I’m beginner not with programming, but with MicroPython. My first problem: i have 3 RP Pi Pico 2040 of the same art. All of them has 
good function with my first prog „led2.py“ (blinking LED), but only one of the three rp2040 has good function with the prog „bme280“. The other two has no function with this. Thonny gives für both the failure: „ line 3 in <module>„ and „ImportError: no module named „bme280“ ".
I had prepared the „good one“ at first and the two „bad ones“ after. All of them sitting in change in the same board. Whats the reason for  not function on the two?

The first three lines of the problem-prog:
1- from machine import Pin, I2C
2- from time import sleep
3- from bme280 import BME280
![3xrp2040](https://github.com/user-attachments/assets/d7d0d100-b37e-4ed0-a5e5-466dd2436b07)
[copy23.zip](https://github.com/user-attachments/files/18081126/copy23.zip)

**Comment by robert-hh** [comment]
You have to copy the file bme280.py to each board.
